src/compiler/stdpar/MallocToUSM.cpp

```diff
@@ -100,9 +100,21 @@ bool isRestrictedToRegularMalloc(llvm::Function* F) {
 }
 
 bool isStdFunction(llvm::Function* F) {
+  static const std::string StdPrefixes[21] =
```

Comment:
```suggestion
  static const std::string StdPrefixes[] =
```
Let the compiler figure out the length?

Reply:
Whoops, I only put the length there explicitly it for me to be sure that I added all prefixes that I wanted to add